src/data_provider/Extractor.py

```diff
@@ -20,14 +33,14 @@ def get_day_to_services_dict(calendar_df: pd.DataFrame):
         return day_to_services
 
     @staticmethod
-    def get_services_list(calendar_df):
+    def get_services_list(calendar_df) -> list:
         services = []
         for service_id, _, _, _, _, _, _, _ in calendar_df.itertuples():
             services.append(service_id)
         return services
```

Comment:
this really needs to be refactored, `return calendar_df.index` or `return list(calendar_df.index)` should be fine